### PR TITLE
Bakta data manager update to tool version 1.8.1 and light db

### PR DESCRIPTION
**Pull request by pimarin**
I update the data manager to include database to the 1.8.1 bakta version

### CODE REVIEW

**Comment by pimarin** [comment]
@bernt-matthias Is it okay ?

File: data_managers/data_manager_build_bakta_database/.shed.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ name: data_manager_bakta
 owner: iuc
 remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/data_managers/data_manager_build_bakta_database
 type: unrestricted
+homepage_url: https://github.com/galaxyproject/tools-iuc/tree/master/data_managers/data_manager_build_bakta_database
```
Comment (bernt-matthias) [inline data_managers/data_manager_build_bakta_database/.shed.yml:11]: Better link to some bakta web site?

Comment (bernt-matthias) [reply data_managers/data_manager_build_bakta_database/.shed.yml:11]: Shed.yml seems incomplete, or?

File: data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc [hunk 2]
```diff
@@ -4,6 +4,6 @@
 # value, dbkey, bakta_version, path
 #
 # for example
-7197299	V0.0_date_test	0.0	${__HERE__}
-V1.0_2022-10-12	7197299	1.4	/tmp/tmpiyh6lcqw/galaxy-dev/tool-data/bakta_database/7197299
-V2.0_2022-11-25	7360139	1.5	/tmp/tmpiyh6lcqw/galaxy-dev/tool-data/bakta_database/7360139
+V1.0_2022-10-12	7197299	1.4	db
```
Comment (bernt-matthias) [inline data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:7]: This does not look right. There are no paths?

File: data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.xml [hunk 3]
```diff
@@ -31,14 +33,20 @@
         <test expect_num_outputs="1">
             <param name="test_data_manager" value="--test"/>
             <param name="database_select" value="1.0"/>
-            <output name="output_file" value="bakta_test_data_manager.json" />
+            <output name="output_file" value="bakta_test_data_manager1.json" />
```
Comment (bernt-matthias) [inline data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.xml:36]: Why not just overwrite `bakta_test_data_manager.json`? At least delete the old fine if not needed anymore.

File: data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.py [hunk 4]
```diff
@@ -98,122 +116,102 @@ class InstallBaktaDatabase(GetBaktaDatabaseInfo):
...
-        else:
-            print(f"Error: corrupt database file! "
-                  f"calculated md5 = {md5.hexdigest()}"
-                  f" different from {self.md5} ")
-
-
-"""
-This is the method to download the amrfinderplus database need by bakta.
-Deprecated to use the amrfinderplus data_manager
-    def update_amrfinderplus_db(self):
```
Comment (bernt-matthias) [inline data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.py:165]: Just to be sure: this function has not been used anyway?

Comment (pimarin) [reply data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.py:165]: No, it was before I made the amrfinderplus datamanager

File: data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.xml (hunk 3 above)
Comment (pimarin) [reply data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.xml:36]: I didn't know it was possible

File: data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc (hunk 2 above)
Comment (pimarin) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:7]: I updated tests

File: data_managers/data_manager_build_bakta_database/.shed.yml (hunk 1 above)
Comment (pimarin) [reply data_managers/data_manager_build_bakta_database/.shed.yml:11]: I add the guthub repo to bakta

**Comment by bernt-matthias** [comment]
Can you cherry-pick the commit from https://github.com/galaxyproject/tools-iuc/pull/5295 .. ?

**Comment by pimarin** [comment]
okay I just confused by the galaxy error with planemo test:
``` 
File "/home/pierre/galaxy/lib/galaxy/tool_util/data/__init__.py", line 675, in _add_entry
f"Attempted to add fields ({fields}) to data table '{self.name}', but this entry already exists and allow_duplicates is False."
galaxy.exceptions.MessageException: Attempted to add fields (['V1.0_2022-10-12', '7197299', '1.4', '/home/pierre/galaxy/tool-data/bakta_database/7197299']) to data table 'bakta_database', but this entry already exists and allow_duplicates is False.
```

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**